src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/Et1VettingService.java

```diff
@@ -88,7 +93,7 @@ public class Et1VettingService {
     private static final String TRIBUNAL_SCOTLAND = "Scotland";
     private static final String ACAS_CERT_LIST_DISPLAY = "Certificate number %s has been provided.<br><br><br>";
     private static final String NO_ACAS_CERT_DISPLAY = "No certificate has been provided.<br><br><br>";
-
+    static final String DOCGEN_ERROR = "Failed to generate document for case id: %s";
```

Comment:
Why package private?  Prefer explicit private.
If because unit tests reference it, then better to copy for unit tests as they should not dictate how the real code is written